pydap_extras/app.py

```diff
@@ -1,10 +1,19 @@
 import sys
 from paste.httpserver import serve
 from pydap.wsgi.ssf import ServerSideFunctions
-from pydap_extras.handlers import csv
+from pydap_extras.handlers import csv, sql
 
 if __name__ == "__main__":
-    csv._test()
-    application = csv.CSVHandler(sys.argv[1])
+    handler_type = sys.argv[1]
+
+    if handler_type == "csv":
+        csv._test()
+        application = csv.CSVHandler(sys.argv[2])
+    elif handler_type == "sql":
+        sql._test()
+        application = sql.CSVHandler(sys.argv[2])
```

Comment:
This is fine the way it is, but we could make this more flexible and dynamic. Given a `handler_type` from the user we could _just_ import the handler needed using something like [this](https://stackoverflow.com/questions/6677424/how-do-i-import-variable-packages-in-python-like-using-variable-variables-i). From there we seem to be using the same code structure for the tests so we could just make it generic.